docs/Watch.md

```diff
@@ -44,6 +46,13 @@ Cancel the watch task with the watcher, the `onCanceled` will be called after su
 1. The watcher will be removed from [watchers](#watchers-instance) map.
 2. If the [watchers](#watchers-instance) map contain the watcher, it will be moved to [cancelWatchers](#cancelwatchers) and send cancel request to [requestStream](#requeststream-instance).
 
+## requestProgress function
+
+Send the most updated revision number to all active [watchers](#watchers-instance)
```

Comment:
Send the latest revision processed by each watcher to all... ?

Reply:
Changed to be similar to what you suggested